Makefile

```diff
@@ -0,0 +1,51 @@
+SHELL := '/bin/bash'
+SENDGRID := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_email_sendgrid_api_key | jq -r .'SecretString')
+GOVPAY := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_gov_pay_key | jq -r .'SecretString')
+ORDNANCESURVEY := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_ordnance_survey_license_key | jq -r .'SecretString')
+
+.PHONY: all
+all:
+	@${MAKE} up
+
+.PHONY: run
+run:
```

Comment:
should this really be called "run" ?  it effectively runs composer so should it be called compose or something?

Reply:
renamed these to have `dc-` prefixes